tools/vagrant/bootstrap.sh

```diff
@@ -52,11 +52,11 @@ echo "export NRF52_SDK_ROOT=/usr/nrf52-sdk" >> ${HOME}/.bashrc
 
 sudo usermod -aG dialout vagrant
 
-# Make sure we're using Java 8 for Cooja
-sudo update-java-alternatives --set /usr/lib/jvm/java-1.8.0-openjdk-amd64
+# Make sure we're using Java 11 for Cooja
+sudo update-java-alternatives --auto
 
 # Environment variables
-echo "export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64" >> ${HOME}/.bashrc
+echo "export JAVA_HOME=/usr/lib/jvm/default-java" >> ${HOME}/.bashrc
```

Comment:
Cooja does not need JAVA_HOME any longer so you should be able to remove this line.

Reply:
I can take this out and re-test.